ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -11,5 +11,10 @@ ref:
     env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
   - name: "release:latest"
     env: RELEASE_IMAGE_LATEST
+  env:
+  - name: "CLUSTER_VARIANT"
+    default: "default"
+    documentation: |-
+      Set to "single-node" for a cluster with a single control plane node and no workers.
```

Comment:
Documentation should be more general I assume
Because cluster variant can be a lot of things: single-node, mirror, xlarge, compact, rt, etc.

Reply:
It can't - I just made up that variable. What you mention is only relevant for template based jobs, as far as my understanding goes.